src/main/resources/data/us-productized-dataset/us-productized-data.properties

```diff
@@ -65,7 +65,7 @@ log.all.requests.responses=false
 # Identity
 identity.feature.toggle=false
```

Comment:
Maybe we should set it to true by default

Reply:
Done!